src/test/java/uk/ac/ebi/eva/pipeline/jobs/steps/PopulationStatisticsGeneratorStepTest.java

```diff
@@ -56,40 +53,31 @@
 public class PopulationStatisticsGeneratorStepTest {
 
     private static final String SMALL_VCF_FILE = "/small20.vcf.gz";
+    private static final String STATS_FILE_POSTFIX = ".variants.stats.json.gz";
+    private static final String MONGO_DUMP = "/dump/VariantStatsConfigurationTest_vl";
+
+    @Rule
+    public TemporalMongoRule mongoRule = new TemporalMongoRule();
+
+    @Rule
+    public PipelineTemporaryFolderRule temporaryFolderRule = new PipelineTemporaryFolderRule();
 
     @Autowired
     private JobLauncherTestUtils jobLauncherTestUtils;
     @Autowired
     private JobOptions jobOptions;
 
-    private ObjectMap variantOptions;
-    private ObjectMap pipelineOptions;
-    private File statsFile;
-
     @Test
     public void statisticsGeneratorStepShouldCalculateStats() throws IOException, InterruptedException {
```

Comment:
Part of the configuration code for this test group was identical so it was moved to a common function.